os/mounting-storage.md

```diff
@@ -35,11 +37,13 @@ coreos:
         Description=Formats the ephemeral drive
         After=dev-xvdb.device
         Requires=dev-xvdb.device
+        ConditionPathExists=!/var/lib/format-xvdb
```

Comment:
As long as you attach the device before the first reboot, this should be fine (though, I haven't actually tested it myself), so a `nova boot` followed by a `nova volume-attach` should work. That does feel a bit fragile though, so maybe a call to `blkid` to test for a formatted filesystem would be more robust.